deku-derive/src/macros/mod.rs

```diff
@@ -235,18 +235,26 @@ fn gen_field_args(
     bits: Option<&Num>,
     bytes: Option<&Num>,
     ctx: Option<&Punctuated<syn::Expr, syn::token::Comma>>,
+    ctx_nested: Option<&Punctuated<syn::Expr, syn::token::Comma>>,
 ) -> syn::Result<TokenStream> {
     let crate_ = get_crate_name();
     let endian = endian.map(gen_endian_from_str).transpose()?;
     let bits = bits.map(|n| quote! {::#crate_::ctx::Size::Bits(#n)});
     let bytes = bytes.map(|n| quote! {::#crate_::ctx::Size::Bytes(#n)});
     let ctx = ctx.map(|c| quote! {#c});
+    let ctx_nested = ctx_nested.map(|c| quote! {#c});
 
     // FIXME: Should be `into_iter` here, see https://github.com/rust-lang/rust/issues/66145.
-    let field_args = [endian.as_ref(), bits.as_ref(), bytes.as_ref(), ctx.as_ref()]
-        .iter()
-        .filter_map(|i| *i)
-        .collect::<Vec<_>>();
+    let field_args = [
+        endian.as_ref(),
+        bits.as_ref(),
+        bytes.as_ref(),
+        ctx.as_ref(),
+        ctx_nested.as_ref(),
```

Comment:
should be first?

Reply:
I like `MyCtx` being first, since it's "non-default" deku usage and something you want to see first in impl error.